src/openrct2/localisation/StringIds.h

```diff
@@ -3905,6 +3905,7 @@ enum
 
     STR_CREATE_DUCKS_TIP = 6356,
     STR_REMOVE_DUCKS_TIP = 6357,
+    STR_TILE_NEEDS_ATLEAST_ONE_ELEMENT = 6358,
```

Comment:
A small nitpicky thing, since this string is unrelated to the two above please add a white line this one.